viewer/packages/sector-loader/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@reveal/cad-parsers": "workspace:*",
+    "@reveal/logger": "workspace:*",

```

Comment:
The logging is used only for unit test the changes, is possible to have an alternative avoiding logger dependencies!